.github/workflows/bundler.yml

```diff
@@ -22,10 +22,12 @@ jobs:
           # set proper version
           #composer config version --unset
           v=$(echo ${{ github.ref }} | cut -d / -f 4)
-          echo "::set-env name=version::$v"
+          #echo "::set-env name=version::$v"
```

Comment:
we use simillar construction in unit testing: https://github.com/atk4/core/blob/develop/.github/workflows/unit-tests.yml#L43